dt-core/admin/menu/tabs/tab-workflows.php

```diff
@@ -957,6 +971,31 @@ private function sort_workflows_by_name( $workflows ): array {
         return $workflows;
     }
 
+    public function right_column() {
+        ?>
+        <div id="postbox-container-1" class="postbox-container">
```

Comment:
Once the use of `$this->template('right_column')` is updated above, we can remove this container div.